src/config/dynamic-theme-fixes.config

```diff
@@ -1293,6 +1293,19 @@ span[data-testid="file-tree-file__comments"] > span {
 span.code {
     background-color: rgba(255, 247, 218, 0.08) !important;
 }
+[type="button"]:not(.css-1l4j2co, .css-1h2061w, [aria-expanded]) {
+    background-color: rgba(159, 179, 213, 0.24) !important;
```

Comment:
Use something along the lines
```css
background-color: ${rgba(46, 66, 100, 0.24)} !important;
```

Reply:
Thanks for the tip! I shall copy the exact initial color codes then.